50-tools/weblate-create-projects-oneshot.sh

```diff
@@ -0,0 +1,66 @@
+#!/bin/bash
+
+# This script generates initial projects import script. It was used only once
+# and it could be called never more. Once regular projects maintenance will be
+# implemented, it could be deleted. This is just a source of needed code.
```

Comment:
What specifically do you need the script for? Why is it not enough to just create a new project on the Weblate Web interface?

Reply:
@sknorr If it's simpler to set things up in Weblate Web interface, I think we can trash it. I found it in my research for projects translated via Weblate and I imported it in my repo for future evaluation.

Reply:
@sknorr I fixed your comments above for now. However, I can drop it from the PR if you want.

Reply:
weblate-create-projects-oneshot.sh script creates initial setup for large Weblate projects with many projects and/or subprojects.

I submitted the weblate-create-projects-oneshot.sh script to several projects to document the process of the initial import. However it is no more useful for the project itself, the code can be recycled to create a new large project with many subprojects.

Looking at your project, it would be very useful to use such script, as it will save hours of clicking. Additionally, you would probably need a script to update projects. List of translation domains will probably change over the time, and it is always simpler (for me) to call a script instead of comparing lists manually.

You can add it to the pull request, and I will update it once the initial import will succeed, or you can mail it to me, and I will make a pull request with the script that worked.

However you can write and push such script, you cannot run it. To use it, you need openSUSE Heroes account, openSUSE VPN access, root access to the Weblate server and admin privileges in the Weblate. I can do run it for you.

Please decide about a structure of the Weblate project:

- You can have one project doc-sle with many components:
doc-sle/adm_support-develop
doc-sle/adm_support-sle-15-sp1
doc-sle/adm_wbem-develop
doc-sle/adm_wbem-sle-15-sp1

- Or you can have many projects, each with few components.
doc-sle-adm_support/develop
doc-sle-adm_support/sle-15-sp1
doc-sle-adm_wbem/develop
doc-sle-adm_wbem/sle-15-sp1

From translators' perspective, there is only a visual difference. But there is a big difference in behavior: components of one project share equal strings. If there will be many projects setup, only components inside one project share equal strings.

weblate-ssh is just a wrapper that calls correct ssh command to connect to the Weblate server. It was not published as it depends on openSUSE intranet changes. I will consult whether I can publish it, as it contains sensitive information. But anyway, you have not the correct SSH key to get in. Use of Django Shell works around limitations of Weblate API (in time of writing the script).